bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultPropertyKey.java

```diff
@@ -0,0 +1,28 @@
+/**
+* Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*/
+package org.eclipse.smarthome.core.thing;
+
+/**
+ * The {@link DefaultPropertyKey} enumeration defines all keys for default properties of a {@link Thing}.
```

Comment:
Reading this after reviewing your code I think we might better rename this to "CommonPropertyKey" or so. The word "default" somehow implies that every thing has to have these properties, which is not true. We just "suggest" these keys IF a thing has a property of a certain "common" meaning.